src/privileged/FirefoxMonitor.jsm

```diff
@@ -165,9 +165,32 @@ this.FirefoxMonitor = {
 
     this.domainMap.clear();
     sites.forEach(site => {
+      // Round the PwnCount:
+      // If < 100k: keep as is; e.g. 12,345 -> 12,345
+      // If < 1M: round to nearest 100k; e.g. 234,567 -> 200,000
+      // If < 1B: round to nearest millions; e.g. 123,456,789 -> 123 million
+      // If >= 1B: round to nearest billions; e.g. 9,123,456,789 -> 9 billion
```

Comment:
Not really "nearest" though, right?
If we have 9,999,903 breached accounts, wouldn't the code round that down to "9 million"?